discussion/templates/discussion/discussion.html

```diff
@@ -805,7 +827,7 @@
     method: "POST",
     data: {
       id: c.attr('data-id'),
-      text: open_editors[c.attr("data-id")].getContentsAsCommonMark()
+      text: c.find('.comment-edit textarea').val()
```

Comment:
Do we have a risk here of someone entering Javascript into the comment section and we just grab it? How do we make sure there is no dangerous Javascript in the comments?

Reply:
From a security standpoint this PR doesn't change anything since we're looking at client-side code.

My usual stance on unstructured user input is that it should be sanitized before it is displayed. Discussion comments are treated as CommonMark and are rendered using the CommonMark library with the "safe" flag turned on, which strips out raw HTML and unsafe link destinations.

Reply:
Thanks.